src/Components/CriticalCareRecording/PressureSore/CriticalCare__PressureScoreViewer.res

```diff
@@ -0,0 +1,11 @@
+@react.component
+export make = (~pressureSoreParameter, ~id, ~consultationId) => {
+  let pressureSoreParameter = CriticalCare__PressureSore.makeFromJsx(pressureSoreParameter)
+  <CriticalCare__PressureSoreEditor
```

Comment:
btw what is difference between pressure score and pressure sore? 😅 
because file is named as `...Score...res` and other places have `sore`

This component is almost seems blank. can we avoid this file?

Reply:
Filename was a typo mistake. I have fixed it
This component is not blank. It is used for showing Preview Mode in Updates Section